AzureImageSchema: Override hyperv_generation

kamalca opened the pull request:
Returns LISA to behavior previous to #3948 with regards to hyperv_generation.

squirrelsc reviewed (dismissed):
Please hold this, until we have a conclusion.

kamalca committed:
Temporary fix for hyperv_generation regression

Behavior was changed which caused LISA to respect hyperv-generation set in runbook. Reverting this change to unblock workflows.

LiliDeng reviewed (changes requested):
wait for final solution
